src/storage/mvcc/txn.rs

```diff
@@ -262,6 +273,19 @@ impl<S: Snapshot> MvccTxn<S> {
         let ts = self.start_ts;
         self.put_write(key, ts, write.to_bytes());
         self.unlock_key(key.clone());
+        if self.collapse_rollback {
```

Comment:
Seems we could also collapse the previous rollback for the case when receives `Rollback` before `Prewrite`?

Reply:
I don't think it is necessary, don't over-optimize.

Reply:
Done